cas-server-webapp/src/main/webapp/WEB-INF/spring-configuration/applicationContext.xml

```diff
@@ -68,4 +68,25 @@
             value-ref="serviceTicketUniqueIdGenerator" />
     </util:map>
 
+    <bean id="passThroughController" class="org.springframework.web.servlet.mvc.UrlFilenameViewController"/>
```

Comment:
Why did you move the controller and handling in the application context? It fits more in the servlet context, doesn't it?